rakelib/artifacts.rake

```diff
@@ -165,15 +165,15 @@ namespace "artifact" do
     when "darwin"
       build_tar(*license_details, platform: "-darwin-#{arch}")
     end
-    system("./gradlew deleteLocalJdk -Pjdk_bundle_os=#{os_name}")
+    raise "Gradle failed to execute deleteLocalJdk" unless system("./gradlew deleteLocalJdk -Pjdk_bundle_os=#{os_name}")
   end
 
   # Create an archive pack using settings appropriate for the running machine
   def create_local_archive_pack(bundle_jdk)
     @bundles_jdk = bundle_jdk
-    system("./gradlew copyJdk") if bundle_jdk
+    raise "Gradle failed to execute copyJdk" unless system("./gradlew copyJdk") if bundle_jdk
```

Comment:
I wonder if it would be better to split this for readability, rather than having `unless` and `if` on the same line

```suggestion
    if bundle_jdk
      raise "Gradle failed to execute copyJdk" unless system("./gradlew copyJdk") 
    end  
```

Reply:
I agree! Thanks for noting it